### PR TITLE
Add-some-Unit-Tests

### PR DESCRIPTION
**Pull request by TheRealHaui**
I added some Unit Tests to increase code coverage which I want to contribute.

### CODE REVIEW

File: commons-numbers-combinatorics/src/test/java/org/apache/commons/numbers/combinatorics/FactorialDoubleTest.java [hunk 1]
```diff
@@ -121,10 +124,31 @@ public void testCacheDecrease() {
...
         }
+
         return result;
+
+
     }
+
+  @Test
+  public void testWithCacheThrowsCombinatoricsException() {
+
```
Comment (ameyjadiye) [inline commons-numbers-combinatorics/src/test/java/org/apache/commons/numbers/combinatorics/FactorialDoubleTest.java]: Don't keep empty lines. by guidelines after method name no empty line should be there neither before ending method.

File: commons-numbers-arrays/src/test/java/org/apache/commons/numbers/arrays/SafeNormTest.java [hunk 2]
```diff
@@ -24,39 +26,52 @@
...
             n += v[i] * v[i];
         }
         final double expected = Math.sqrt(n);
-        Assert.assertEquals(expected, SafeNorm.value(v), 0d);
+        assertEquals(expected, SafeNorm.value(v), 0d);
+    }
+
+    @Test
+    public void testValueReturningPositive() {
+
```
Comment (ameyjadiye) [inline commons-numbers-arrays/src/test/java/org/apache/commons/numbers/arrays/SafeNormTest.java]: Say no to empty lines

File: commons-numbers-combinatorics/src/test/java/org/apache/commons/numbers/combinatorics/LogFactorialTest.java [hunk 3]
```diff
@@ -115,4 +116,20 @@ private double logFactorial(final int n) {
         }
         return logSum;
     }
+
+    @Test
+    public void testValueThrowsCombinatoricsException() {
+
```
Comment (ameyjadiye) [inline commons-numbers-combinatorics/src/test/java/org/apache/commons/numbers/combinatorics/LogFactorialTest.java]: Same here, no empty line

**Comment by TheRealHaui** [comment]
@ameyjadiye 
Removed evil empty lines in pull request #9.
Don't know why Github didn't add the change to the original pull request.  
Anyway, I think that doesn't bother a lot.

**Comment by ameyjadiye** [comment]
Thanks @TheRealHaui. will take another round or review.

If you wish that new code to be come under same PR you have to push to same branch :wink: 
for #8 you pushed to ```TheRealHaui:master``` while for #9 you pushed to ```TheRealHaui:Add-some-Unit-Tests```.

**Comment by TheRealHaui** [comment]
Yes ...
Forgot that I created a branch explicitly yesterday.
After pughing ...
Think that doesn't bother a lot, simply delete the pull request you do not need.

**Comment by ameyjadiye** [comment]
Yes, since you have opened it, you have right to close #9 , but simply push those changes to your master so those will be available in #8. Thanks.

**Comment by TheRealHaui** [comment]
You're right, that's the most simple way.
Pushed changes now to master.
They appear here in this pull request.
Furthermore I will close pull request #9.

**Comment by ericbarnhill** [comment]
Thanks for the contributions. Regarding empty lines and such. We require that the submission cleanly passes maven's checkstyle. This means when you run `mvn site`, or even just `mvn checkstyle:checkstyle`, the checkstyle report under reports should be clean. If you check that I think it will catch things like empty lines. Let me know if you need any help with this and apologies if you already knew this.

**Comment by aherbert** [comment]
This PR is out-of-date. Please open a new PR if you find any issues.

**Pull request closed without merging**